---
layout: page
title: About me
---

I am a population geneticist and bioinformatician. I strive to develop novel computational methods that enable more comprehensive and equitable genomics research. 

I received my Ph.D. from the Georgia Institute of Technology in Quantitative Biosciences under the guidance of [Joseph Lachance](https://popgen.gatech.edu/). My doctoral research, *Theoretical and Empirical Population Genetics of Admixture and Introgression*, spanned topics from inferring the present-day fitness of Neanderthal introgressed sequence in admixed genomes to estimating the ratios of females and males from different ancestries that contributed to admixed populations. I also contributed to several papers elucidating the genetic basis of prostate cancer and male-pattern baldness in men of African descent, among others.

Currently, I am a Postdoctoral Researcher at the [Akey Lab](https://akeylab.princeton.edu/), Princeton University. My postdoctoral work centers on developing novel computational methods for the comprehensive analysis of structural variation through de novo genome assembly and pangenomics. I recently developed a new method that improves variant phasing by leveraging methylation information of long-read sequencing data, and I am now working on incorporating the benefits of methylation information into de novo genome assemblies. 

Broadly, my research aims to develop novel population and statistical genetics frameworks for studying structural variants and pangenomes. Because structural variants affect more genomic sequence than single-nucleotide variants, they often have stronger functional and evolutionary effects. Capturing these dynamics requires new population genetic models that account for the unique properties of structural variants, such as non-constant mutation rates and effective population sizes, while minimizing reference bias. Integrating these approaches with pangenomes and ancient DNA offers an unprecedented opportunity to explore what makes us uniquely human and to uncover the contribution of structural variation to complex phenotypes, such as cardiovascular diseases.
